modules/ROOT/pages/sandboxes.adoc

```diff
@@ -0,0 +1,39 @@
+= Sandboxes
```

Comment:
Can we make this title more robust? What, specifically, ABOUT sandboxes that the user wants to know to achieve a benefit?